drafts/str4d-overwinter-activation/draft1.rst

```diff
@@ -0,0 +1,232 @@
+::
+
+  ZIP: ???
+  Title: Network Upgrade Mechanism
+  Author: Jack Grigg <jack@z.cash>
+  Comments-Summary: No comments yet.
+  Category: Process
+  Created: 2018-01-08
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be interpreted as
+described in RFC 2119.
+
+The following terms are interpreted as follows:
+
+Branch
+  A chain of blocks with common consensus rules, where the first block in the chain is not the genesis block,
+  but the child of a parent block created under an older set of consensus rules.
+
+Hard fork
+  The creation of a new branch by a change in the consensus rules of the network. Nodes that do not recognize
+  the new rules will continue to follow the old branch.
+
+Network Upgrade
+  An intentional hard fork undertaken by the community in order to improve the network.
+
+
+Abstract
+========
+
+This proposal defines a mechanism for coordinating upgrades of the Zcash network, in order to remove ambiguity
+about when network upgrades will activate, provide defined periods in which users should upgrade their local
+software, and minimize the risks to both the upgrading network and those users opting out of the changes.
+
+
+Motivation
+==========
+
+Zcash is a *consensual currency*: nobody is ever going to force someone to use a specific software
+implementation or a specific branch of Zcash. [#consensual-currency]_ As such, different sub-communities will
+always have the freedom to choose different variants or branches which offer different design trade-offs.
+
+The current Zcash software includes an *auto-senescence* feature, causing nodes running a particular version
+to automatically shut down 16 weeks after that version was released. This was implemented for several reasons
+[#release-lifecycle]_:
+
+- It gives the same systemic advantage of removing old software as auto-upgrade behavior.
+
+- It requires users to individually choose one of the following options:
+
+  - Upgrade to a more recent software release from the main network.
+
+  - Upgrade to an alternative release.
+
+  - Ignore the "deprecation" and keep running the older software.
```

Comment:
You said it shuts down, so how can the deprecation be ignored? You mean switching to even older version without auto-senescence.
When reading this the thought comes to mind - "what if developers don't issue a new version within 16 weeks. So the network shuts down?"

Reply:
I'm referring to the `-disabledeprecation` flag that currently exists.

Reply:
I'll alter this to say "Modify their node in order to keep running the older software", which is clearer as to the action the user is taking.

Reply:
perhaps add "e.g. using the `-disabledeprecation` flag"?

Reply:
hence the "e.g." in any case, the phrasing is confusing; if you don't want to say how , would at least change "ignore" to "circumvent"

Reply:
@arielgabizon I've already change this to say `Modify...` instead of `Ignore...`, as above.